.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe this should be added to gitignore :) to have a valid package you only need a package.swift file and right files structure conforming to the package description.

Reply:
the Package.resolved file is under this directory I believe.

It stores the hash values for your resolved packages. Apple recommends keeping that file in your repo.

Reply:
Nah, .resolved usually stays next to the package.swift file in the root when you create a package from scratch with 'swift package init' and then add some dependencies.

Reply:
If there's no special Xcode specific configuration it can be removed from repo. It's Xcode file anyway, and spm is cross-platform-like thing and can be used on Linux where you won't have Xcode :)